runtime/queries/javascript/injections.scm

```diff
@@ -9,6 +9,14 @@
   ]
   arguments: (template_string) @injection.content)
 
+; Parse the contents of gql template literals
+
+((call_expression
+   function: (identifier) @ignore
```

Comment:
Do we need to do this `@ignore`?

Reply:
I think something like `@_template_function_name` or something else beginning with `_` would fit well with other injections :+1:

Reply:
I'm not sure! This is from what @the-mikedavis posted earlier in this PR: https://github.com/helix-editor/helix/pull/1515#issuecomment-1013771472